src/hotspot/share/interpreter/templateInterpreter.cpp

```diff
@@ -104,6 +104,19 @@ EntryPoint::EntryPoint(address bentry, address zentry, address centry, address s
   _entry[vtos] = ventry;
 }
 
+EntryPoint::EntryPoint(address aentry, address ientry, address lentry, address fentry, address dentry, address ventry) {
+  assert(number_of_states == 10, "check the code below");
+  _entry[btos] = ientry;
+  _entry[ztos] = ientry;
+  _entry[ctos] = ientry;
+  _entry[stos] = ientry;
```

Comment:
next patch is to get rid of these entry points since I think nothing branches to them.

Reply:
Thanks for reviewing. 

I'll file an RFE to keep at this. I didn't want to keep pulling at this thread for this RFE since it seems like quite a chunk of code will be unravelled here.